lib/Doctrine/Migrations/Finder/GlobFinder.php

```diff
@@ -12,12 +12,12 @@
     /**
      * @return string[]
      */
-    public function findMigrations(string $directory, ?string $namespace = null) : array
+    public function findMigrations(string $directory) : array
```

Comment:
As discussed, please reintroduce the `$namespace` argument as optional and if it is provided, filter the classes returned to only be the ones in that namespace.